release-1.8/release_notes_draft.md

```diff
@@ -44,3 +44,37 @@ please check out the [release notes guidance][] issue.
 ## **Deprecations**
 
 ## **Notable Features**
+
+### **Node Components**
+#### kubelet
```

Comment:
Need to add the new alpha CPU manager https://github.com/kubernetes/features/issues/375 to this list


```
[alpha] Add support for workload placement control with kubelet CPU manager. ([#375](https://github.com/kubernetes/features/issues/375), [@sjenning](https://github.com/sjenning), [@ConnorDoyle](https://github.com/ConnorDoyle)
```

@ConnorDoyle @derekwaynecarr 

Connor, that short description sound good?